### DISCUSSION TITLE
How to enable WSLG for existing distro with the latest WSL(G) version?

### DISCUSSION THREAD

**Discussion by MaksymSofer** [microsoft/wslg#621] [Q&A]
In the last WSL version --update option open Microsoft Store instead of making any changes to the system.
I have one distro which i need to enable wslg  - how to do it?
C:\WINDOWS\system32>wsl --status
Default Distribution: Ubuntu-20.04
Default Version: 2
WSL version: 0.50.2.0
Kernel version: 5.10.74.3
WSLg version: 1.0.29
Windows version: 10.0.22000.376

**Comment by ghost** [comment]
WSLg is automatically enabled for all WSL2-based distros afaik. you should be able to see if wslg is installed by checking the existence of `/mnt/wslg` and it's files

As long you don't disable it yourself. `guiApplications=false` in INI file `C:/Users/<user>/.wslconfig`
